orchestrator_client.gemspec

```diff
@@ -15,6 +15,6 @@ Gem::Specification.new do |s|
   s.license       = "Apache-2.0"
   s.require_paths = ["lib"]
 
-  s.add_dependency 'faraday', '~> 1.4'
-  s.add_dependency 'net-http-persistent', '~> 4.0'
+  s.add_dependency 'faraday', '~> 2.0'
```

Comment:
It will be easier for us to take this up if we allow both 1.x and 2.x in the mean time so we dont have to immediately change bolt packaging.

Reply:
I adjusted the commits accordingly.